<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppWindow</class>
 <widget class="QMainWindow" name="AppWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Study Group Former</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="courseNameComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>380</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="courseNumberComboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>460</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="UserProfile">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>View User Profile</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="startTimeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="courseNameLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
   <widget class="QLabel" name="courseNumberLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>121</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="startTimeLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Time:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateOfStudyWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Study Group Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni 72 Oldstyle</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="listOfAllGroups">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>561</width>
      <height>341</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string/>
     </property>
    </row>
   </widget>
   <widget class="QPushButton" name="getGroupInfo">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Group Info</string>
    </property>
   </widget>
   <widget class="QPushButton" name="refreshButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateGroup">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Group</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="courseDescriptionTextBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>411</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Group</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>835</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
